host/Cargo.toml

```
@@ -4,6 +4,8 @@ version = "0.1.0"
 edition = "2021"
 
 [dependencies]
+alloy-sol-macro = { version = "0.4" }
+alloy-sol-types = { version = "0.4" }
```

```suggestion
alloy-sol-types = "0.4"
```

I don't think the `alloy-sol-macro` is used directly.